src/components/Footer.jsx

```diff
@@ -0,0 +1,93 @@
+import React from "react";
+import projectLogo from "../assets/projectLogo.svg";
+import { FaFacebook, FaInstagram, FaTwitter, FaGithub } from "react-icons/fa";
+import { GiCheckMark } from "react-icons/gi";
+import { Link } from "react-router-dom";
+import {
+  HOME_ROUTE,
+  ABOUT_ROUTE,
+  BOOKS_ROUTE,
+  FAVOURITES_ROUTE,
+} from "../routes";
+
+function Footer() {
+  return (
+    <footer>
+      <div className="p-10 bg-gray-200 text-black">
+        <div className="max-w-7xl mx-auto">
+          <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-2 gap-x-40 ">
+            <div className="mb-5 flex flex-col justify-around items-center">
+              <img
+                className="mb-11 lg:mr-36"
+                src={projectLogo}
+                alt="checkmark"
+              />
+              <div className="flex flex-col justify-center items-center md:items-start md:text-left">
+                <p className="font-bold text-lg md:self-center lg:self-start">
+                  Contact Us
+                </p>
+                <p>Email Address: example@example.com</p>
+                <p>Phone Number: 0770 000 00 00</p>
+              </div>
+            </div>
+            <div className="flex flex-col">
+              <div className="flex justify-center mt-2">
+                <Link to={HOME_ROUTE} className="mx-5">
```

Comment:
Please make these look like a link in the UI. Because people wouldn't know this is a link unless they hover over it.